.github/workflows/main.yaml

```diff
@@ -33,8 +33,10 @@ jobs:
 
       - run: pip install .
 
+      - run: pip install pytest
+
       - name: run tests
-        run: python setup.py test
```

Comment:
Now that setup.py is gone, how should we release? Can you also update the workflow declared in upload-to-pypi.yaml?